### DISCUSSION TITLE
关于序列推荐中数据预处理问题

### DISCUSSION THREAD

**Discussion by kuzma-long** [RUCAIBox/RecBole#1241] [Q&A] [answered]
序列推荐中对数据处理时，构建用户的交互序列时，我看有的数据集是把rating大于等于3的算作正反馈，有的是只要有交互就算做正反馈，这两种到底哪个更好？

**Comment by hyp1231** [comment, marked as answer]
我觉得可能没有哪种更好之分，要根据具体问题场景判断。

比如如果是评分数据集，如果想探究用户下一个喜欢的商品，自然可能要筛掉低分 rating；如果想探究用户下一个可能感兴趣的商品，可能不筛选好一些。无论怎么处理，只要满足目标场景，方法之间的对比公平，或者直接遵循前人设置都是可以的。
